content/rs/databases/import-export/replica-of/create.md

```diff
@@ -8,118 +8,125 @@ categories: ["RS"]
+            If you regenerate the password, replication to existing destinations fails until their credentials are updated with the new password.
 
-        {{< note >}}
-For a source database on a different RS cluster,
-you can [compress the replication data]({{< relref "/rs/databases/import-export/replica-of/#data-compression-for-replica-of">}}) to save bandwidth.
-        {{< /note >}}
+        1. In the destination database, paste the URL of the source endpoint to the **Replica Of** edit box.  
+        
+        Use the **Save** button to save your changes.  
+        ![Save](/images/rs/icon_save.png#no-click "Save")
 
-    - For a source database in an OSS Redis cluster - Enter the URL of the source endpoint in the format:
+        For source databases on different clusters, you can [compress replication data]({{< relref "/rs/databases/import-export/replica-of/#data-compression-for-replica-of">}}) to save bandwidth.
+        
+    - For a source database on an open source Redis (OSS) cluster - Enter the URL of the source endpoint in one of the following formats:
 
-        - If the database has a password -
+        - For databases with passwords:
 
             ```sh
-            redis://:<redis_password>@<hostname>:<database_port>
+            redis://:<password>@<host>:<port>
             ```
 
             Where the password is the Redis password represented with URL encoding escape characters.
 
-        - If the database has no password -
+        - For databases without passwords: -
 
             ```sh
-            redis://<hostname>:<database_port>
+            redis://<host>:<port>
             ```
 
-{{< note >}}
-If you used the mDNS protocol for the cluster name (FQDN),
-make sure that the [client mDNS prerequisites]({{< relref "/rs/networking/mdns.md" >}}) are met.
-{{< /note >}}
+For best results when using the [Multicast DNS](https://en.wikipedia.org/wiki/Multicast_DNS) (mDNS) protocol to resolve the fully-qualified domain name (FQDN) of the cluster, verify that your client connections meet the [client mDNS prerequisites]({{< relref "/rs/networking/mdns.md" >}}).
 
-## Configuring TLS for Replica Of traffic on the destination database
+## Configure TLS on replica database
 
 When you enable TLS for Replica Of, the Replica Of synchronization traffic uses TLS certificates to authenticate the communication between the source and destination clusters.
+
 To encrypt the Replica Of synchronization traffic, you must also [configure encryption for the source database](#configuring-encryption-of-replica-of-traffic-on-the-source-database).
 
 To enable TLS for Replica Of in the destination database:
 
-1. Hover over the URL of the source endpoint and click ![Unencrypted](/images/rs/icon_unlocked.png#no-click "Unencrypted"):
+1. Select the **Enable TLS Authentication** button  
```

Comment:
```suggestion
1. Select the **Enable TLS Authentication** button:
```

Reply:
Updated, though I didn't use a colon (due to the colon on the previous line).